wallet-connect/dapp.md

```diff
@@ -0,0 +1,142 @@
+# DApp Integration
+
+Trust uses a forked version of WalletConnect with aditional methods to allow **dApp** developers
+to sign transactions for any blockchain. 
+
+## Getting started
+To use Trust's WalletConnect implementation, you need to clone the following repository:
+
+```bash
+git clone git@github.com:TrustWallet/walletconnect-monorepo.git
```

Comment:
is it required to have forked version of trust to use `trust_sign` ?

Reply:
Yes. It is. I didn't submit the pull request to walletconnect.